save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityTagsComponent.kt

```diff
@@ -118,6 +120,7 @@ val vulnerabilityTagsComponent: FC<VulnerabilityTagsComponentProps> = FC { props
                 classes = "rounded-pill text-sm btn-sm mx-1 mt-2"
             ) {
                 if (tagEditable && !props.isEditDisabled) {
+                    // TODO: use translation with substitution
```

Comment:
why not now?

Reply:
I would like to do it as external PR as it might take some time - some `Translation` refactoring might be needed